piqueserver/config/config.toml

```diff
@@ -38,6 +38,13 @@ tips_frequency = "5min"
 # set this if the server should register on the master server list
 master = false
 
+# configure multiple master servers here
+# NOTE: you should leave at least buildandshoot for your server
+# to be viewable by everyone
+master_hosts = [
```

Comment:
I would comment this out my default so that we can can change the defaults in future updates without having to make everyone change their configs.

The further thing worth debating is whether we want to feature this so prominently in the settings as few users will want to change this and people changing it not knowing the ramifications may increase fragmentation.

Reply:
Fair point, it might be better to remove it from the example config entirely. Would definitely avoid users asking for help in not understanding why their server isn't showing up on the masterlist because they changed it by accident.